WPT synchronize css/css-conditional & css/cssom

annevk opened the pull request:
#### 47db5ad7169a4ea04cd044086a5883c793995d94
<pre>
WPT synchronize css/css-conditional &amp; css/cssom
<a href="https://bugs.webkit.org/show_bug.cgi?id=267033">https://bugs.webkit.org/show_bug.cgi?id=267033</a>

Reviewed by Tim Nguyen.

Up to and including:
<a href="https://github.com/web-platform-tests/wpt/commit/714dcd5574a8ea8c9e9063d6030adf662a1df9b9">https://github.com/web-platform-tests/wpt/commit/714dcd5574a8ea8c9e9063d6030adf662a1df9b9</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-048-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-048-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-file-selector-button-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-file-selector-button.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-placeholder-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-placeholder.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/js/CSS-supports-CSSStyleDeclaration-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/js/CSS-supports-CSSStyleDeclaration.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-pseudo-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-pseudo.html:

Canonical link: <a href="https://commits.webkit.org/272607@main">https://commits.webkit.org/272607@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2798c26f123b3948175c26277609d032efd001ef

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/32350 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/11087 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/34173 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34905 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/29282 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13441 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/8296 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28826 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32768 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9352 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28912 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/8141 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/8284 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/28843 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/36246 "Built successfully") | 
| [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/27791 "Build is in progress. Recent messages:") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29413 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/29271 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34407 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8412 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6364 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/32269 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/10071 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/28617 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7540 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/9041 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8976 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Please add upstream commit to commit message

annevk committed:
WPT synchronize css/css-conditional & css/cssom
https://bugs.webkit.org/show_bug.cgi?id=267033

Reviewed by Tim Nguyen.

Up to and including:
https://github.com/web-platform-tests/wpt/commit/714dcd5574a8ea8c9e9063d6030adf662a1df9b9

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-048-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-048-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-file-selector-button-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-file-selector-button.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-placeholder-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-placeholder.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-047-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/js/CSS-supports-CSSStyleDeclaration-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/js/CSS-supports-CSSStyleDeclaration.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-conditional/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-pseudo-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-pseudo.html:

Canonical link: https://commits.webkit.org/272607@main
